<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CK_FileMerger</class>
 <widget class="QDialog" name="CK_FileMerger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CK_FileMerger</string>
  </property>
  <widget class="QLineEdit" name="lineTarget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>601</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTarget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Target</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSource">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSource">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVerify">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Verify</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtSource">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>601</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lstSource">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>601</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
